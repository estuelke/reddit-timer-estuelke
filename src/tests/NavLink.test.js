import React from 'react';
import renderer from 'react-test-renderer';
import StyledNavLink from '../components/NavLink';
import mainTheme from '../styles/themes';

describe('StyledNavLink', () => {
  it('renders correctly with theme passed', () => {
    const component = renderer.create(
      <StyledNavLink theme={mainTheme}>Test Link</StyledNavLink>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('StyledNavLink', () => {
  it('renders correctly without theme passed', () => {
    const component = renderer.create(
      <StyledNavLink>Test Link</StyledNavLink>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
