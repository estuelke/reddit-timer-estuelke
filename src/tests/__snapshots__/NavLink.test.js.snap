// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledNavLink renders correctly with theme passed 1`] = `
<a
  className="sc-AxjAm eKgzTh"
>
  Test Link
</a>
`;

exports[`StyledNavLink renders correctly without theme passed 1`] = `
<a
  className="sc-AxjAm eKgzTh"
>
  Test Link
</a>
`;
